Docker/Dockerfile

```diff
@@ -1,26 +1,46 @@
 FROM library/python:3.7.0-alpine3.8
 
-RUN apk add build-base
-RUN apk add portaudio-dev
-RUN apk add sqlite
-RUN pip install webob
+LABEL maintainer="istvan@codekuklin.com"
```

Comment:
I love labels, this was a very good idea. But keep them consistent with the anki sync server evnironment variables i.e. all uppercase. See https://github.com/ankicommunity/anki-sync-server#envvar-configuration-overrides.

Reply:
I just copied this guide back then https://docs.docker.com/develop/develop-images/dockerfile_best-practices
First, it was **MAINTAINER=** than it became deprecated https://docs.docker.com/engine/reference/builder/#maintainer-deprecated and we started to use **LABEL maintainer=** and now I can't find it the guide anymore.
But I found this: https://docs.docker.com/config/labels-custom-metadata/
_"Label keys should begin and end with a lower-case letter and should only contain lower-case alphanumeric characters, the period character (.), and the hyphen character (-). Consecutive periods or hyphens are not allowed."_
So if you feel like it will break the consistency, than maybe we should remove it.

Reply:
Ah thanks for providing the links to the documentation. 

It's interesting because they don't provide any reasons for their recommendations. But I think we should follow the Docker conventions until there is a reason not do. So I'll consider this issue resolved.